src/clj/chronograph/db/time_span.clj

```diff
@@ -0,0 +1,75 @@
+(ns chronograph.db.time-span
+  (:require [next.jdbc.sql :as sql]
+            [chronograph.db.core :as db]
+            [chronograph.utils.time :as time]))
+
+(defn create!
+  [tx timer-id]
+  (let [now (time/now)]
+    (sql/insert! tx
+                 :time-spans
+                 {:id (java.util.UUID/randomUUID)
```

Comment:
This is unnecessary if you create a default value for the id in the database

Reply:
Done. PG now generates UUID for us.